README.md

```diff
@@ -3,19 +3,25 @@
 # MOC
 This document describes the Multi-Order Coverage map method (MOC) to specify arbitrary coverages for sky regions and/or time coverages and potentially other dimensions. The goal is to be able to provide a very fast comparison mechanism between coverages. The mechanism is based on a discretization of space and time dimensions. The system is based on the definition of a specific storage of the map coverage using predefined cells hierarchically grouped which makes it easy to produce and use for exploring astronomical collections. There are already a few applications and libraries which are taking advantage of this new standard.
 
-# Status
+## Document Status
 
-Under development.
+### Latest Stable
 
-# Working on this Document
+The current IVOA Recommendation is [REC-2.0](https://www.ivoa.net/documents/MOC/index.html).  Prior versions are available on the [IVOA Documents page](https://www.ivoa.net/documents/index.html).
```

Comment:
With current repo storage tree I think the link will end up being:
https://ivoa.net/documents/MOC/20220317/index.html

once the date gets fixed

Reply:
Yes, that link should be most reliable.  I've made this change.  I'll wait to merge until the date fix is in place so that I can verify the link works.